kernel/common/ezautolinkoperator.php

```diff
@@ -42,15 +54,52 @@ function formatUri( $url, $max )
         }
         return "<a href=\"$url\" title=\"$url\">$text</a>";
     }
+    
+    static function formatUriCallback( $matches )
```

Comment:
This class method name still leaves something to be desired. We do not think that 'Callback' method name text is clear or needed. We suggest `formatUrlLinkHtml` as a replacement for this class method name.